content/enterprise_influxdb/v1.9/about-the-project/contributing.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Contribute to InfluxDB OSS
+description: Contribute to the InfluxDB OSS project.
+menu:
+  enterprise_influxdb_1_9_ref:
+    name: Contribute to InfluxDB OSS
+    weight: 30
+    parent: About the project
+---
+
+To learn how you can contribute to the InfluxDB OSS project, see
+[Contributing to InfluxDB](https://github.com/influxdata/influxdb/tree/1.8/CONTRIBUTING.md)
```

Comment:
Do we want to link to 2.0 OSS license info here too? If yes, merge after OSS 2.0.7 release.

Reply:
not required. Enterprise is licensed commercially.

Reply:
however, I think this page should be dropped.  Intrepid developers could try and contribute to InfluxDB Enterprise by contributing code to OSS....but, let's just drop this page entirely.